### ISSUE TITLE
backup-appdata will not create any backups when container folder no longer exists in appdata folder

### ISSUE THREAD

**Issue by Beersteddy** [Drazzilb08/userScripts#31]
How to recreate the issue on unraid:
Install any docker app. For example, Prowlar, name it `prowlar-test` (name itself is not important)
Once installed, stop `prowlar-test`
Go to app data and manually **delete `prowlar-test` folder**
Don't do anything to the app.
Run the backup-appdata script

Check backup-data config file, for the purpose of the test, move `prowlar-test` higher up on the stop-list, like in the config below.

The script will run until it reaches `prowlarr-test` on the list and then it will stop making backups of any containers that are after the `prowlarr-test` on the list

From the example config below, only `radarr.tar` will be created, `sonarr.tar` will not be as the script stops.

Config:
```
destination_dir='/mnt/user/BACKUPS'
keep_backup=10
compress=false
exclude_file='/mnt/user/appdata/scripts/backup-appdata/exclude-file.conf'
add_to_stop=true
add_to_no_stop=false
dry_run=false
quiet=false

#------------- DEFINE CONTAINERS TO BACKUP -------------#

stop_list=(
	radarr
	prowlar-test
	sonarr
)

no_stop_list=(
    unpackerr
)

exclusion_list=(
    plex
)

appdata_dir1="/mnt/user/appdata"
appdata_dir2="/mnt/cache_nvme/appdata"

```

log:

```
Script Starting Mar 20, 2023  21:21.17

Full logs for this script are available at /tmp/user.scripts/tmpScripts/Unraid - Backup appdata/log.txt

Config file: /mnt/user/appdata/scripts/backup-appdata/backup-appdata.conf
-----------------------------------
Source Directory: /mnt/user/appdata/radarr
Checking space requirements... Please wait...
Checking space requirements complete...
Backing up radarr
Exclude file /mnt/user/appdata/scripts/backup-appdata/exclude-file.conf
backup file: /mnt/user/BACKUPS/2023-03-20@21.21/radarr.tar

Backup of radarr complete
Container: radarr Backup size: 47M
-----------------------------------
Source Directory: /mnt/user/appdata/prowlarr-test
Container prowlarr-test is not running
Checking space requirements... Please wait...
du: cannot access '/mnt/user/appdata/prowlarr-test': No such file or directory
/mnt/user/appdata/scripts/backup-appdata/backup-appdata.sh: line 57: [: : integer expression expected
Checking space requirements complete...
/mnt/user/appdata/scripts/backup-appdata/backup-appdata.sh: line 168: cd: /mnt/user/appdata/prowlarr-test/..: No such file or directory
Script Finished Mar 20, 2023  21:22.18

Full logs for this script are available at /tmp/user.scripts/tmpScripts/Unraid - Backup appdata/log.txt
```

**Comment by Drazzilb08** [comment]
The issue is the existence of the folder. Some error handling needs to be added that if the folder doesn't exist to skip the backup.

**Comment by Drazzilb08** [comment]
Fixed with [6bb32c0](https://github.com/Drazzilb08/userScripts/commit/6bb32c0e9ff609dcd1f43f679bcfc5c3b7e02580)
